### DISCUSSION TITLE
How to lower nitro version in nuxt

### DISCUSSION THREAD

**Discussion by lxccc812** [nitrojs/nitro#2279] [Q&A]
nuxt: 3.9.0
nitro: 2.9.4
node:16.16.0
Nitro error caused by the node version being too low, but I cannot upgrade node (the production environment cannot be upgraded temporarily). I want to lower the nitro version. How should I do it?
Delete node_modules/ Delete pnpm-lock.yaml and specify "nuxt": "3.8.2", still download nitro2.9.4
![image](https://github.com/unjs/nitro/assets/110974969/8da02b20-2ff4-45b0-8efc-e0da2528beb9)

**Comment by SoslanZS** [comment]
Good afternoon. Did you manage to solve this problem?

**Reply by lxccc812** [reply]
I didn't solve this problem. I could only limit the version of nuxt to limit the version of nitro.

**Comment by pi0** [comment]
We can make a workaound for Node.18 support if you are able to share a reproduction of your project.
